docs/DocGenerator/CodeParser.py

```diff
@@ -1,92 +1,147 @@
+"""
+brief: Wazuh DocGenerator code parser.
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+date: August 02, 2021
+license: This program is free software; you can redistribute it
+         and/or modify it under the terms of the GNU General Public
+         License (version 2) as published by the FSF - Free Software Foundation.
+"""
+
 import ast
 import os
 import re
 import json
 import yaml
 from Config import Config
-from TestCaseParser import TestCaseParser
+from PytestWrap import PytestWrap
+from Utils import remove_inexistent
 from docstring_parser import parse
 from comment_parser import comment_parser
 import warnings
+import logging
+
+INTERNAL_FIELDS = ['id', 'group_id', 'name']
+STOP_FIELDS = ['tests','test_cases']
+
 
 class CodeParser:
+    """
+    brief: Class that parses the content of the test files.
+    """
     def __init__(self):
         self.conf = Config()
-        self.test_case_parser = TestCaseParser()
+        self.pytest = PytestWrap()
         self.function_regexes = []
         for regex in self.conf.function_regex:
             self.function_regexes.append(re.compile(regex))
 
     def is_documentable_function(self, function):
+        """
+        brief: Checks if a specific method match with the regexes to be documented.
+        args: -"function (_ast.FunctionDef): Function class with all the information of the method"
```

Comment:
Is there a reason we put the elements of `args` in the same line?

Reply:
No. It´s only because it was a single element. But we can use an extra line if you consider it will be better presented.